charts/castai-agent/values.yaml

```diff
@@ -41,14 +41,18 @@ resources: {}
 additionalEnv:
   PPROF_PORT: "6060"
 
-additionalSecretEnv: {}
-
 # URL to the API server
 apiURL: "https://api.cast.ai"
 
 # Token to be used for authorizing agent access to the API
 apiKey: ""
 
+# Name of secret with Token to be used for authorizing agent access to the API
+# apiKey and apiKeySecretRef are mutually exclusive
+apiKeySecretRef: ""
+
+additionalSecretEnv: {}
```

Comment:
I would move this together with apiKey
These additional env vars would be added to same secret as apiKey